setup.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Open Source Foundries Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import setuptools
+
+with open('README.rst', 'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name='west',
+    version='0.1.8',
+    author='Zephyr Project',
+    author_email='devel@lists.zephyrproject.org',
+    description='Zephyr RTOS Project meta-tool',
+    long_description=long_description,
+    # http://docutils.sourceforge.net/FAQ.html#what-s-the-official-mime-type-for-restructuredtext-data
+    long_description_content_type="text/x-rst",
+    url='https://github.com/zephyrproject-rtos/west',
+    packages=setuptools.find_packages(),
+    classifiers=(
+        'Programming Language :: Python :: 3',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: POSIX :: Linux',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        ),
+    install_requires=(
+        'PyYAML',
+        ),
+    python_requires='>=3',
```

Comment:
In Zephyr  we check for Python 3.4 at configure time. Should we enforce this here in order to benefit from all the improvements between 3.0 and 3.4?